src/commands/init.js

```diff
@@ -24,11 +40,38 @@ function handleInitError(rawError) {
   }
 }
 
+async function handleDatabaseConfiguration() {
+  if (process.env.DATABASE_URL) {
+    return null;
+  }
+
+  logger.success('✅ Checking your database setup');
```

Comment:
If we put your step in a spinner too, we won't need this log.
Let's talk about it to make all step success messages homogeneous.
![image](https://user-images.githubusercontent.com/48474636/81782254-d8020500-94f9-11ea-9e7e-062b92647b5c.png)

Reply:
As said in private, this will be handle in next PR about spinner